.changeset/green-garlics-enjoy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/coin-sui": patch
```

Comment:
a detail but we use minor for everything now, keeping patch for hotfix branches only
```suggestion
"@ledgerhq/coin-sui": minor
```

I know I'm not a big fan too but that's how we do it at the moment